server/controllers/addressController.ts

```diff
@@ -28,9 +28,7 @@ export default class AddressController {
 
     this.auditService
       .sendPageView({
-        userId: res.locals.user.username,
-        userCaseLoads: res.locals.user.caseLoads,
-        userRoles: res.locals.user.userRoles,
+        user: res.locals.user,
```

Comment:
We now have a stronger interface (`HmppsUser`) for what is stored in `res.locals.user` which forces compile time checking that the user has the concept of case loads before trying to access them on the object.  It seemed cleaner to pass the whole `HmppsUser` object through to the audit service methods